package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "publish-to-apple-news",
-  "version": "2.4.5",
+  "version": "2.4.6",
   "license": "GPLv3",
   "main": "index.php",
   "engines": {
-    "node": "16",
-    "npm": "8"
+    "node": ">=18.0.0 <22.0.0",
+    "npm": ">=9.0.0 <11.0.0"
```

Comment:
Is a range necessary?  We had a problem on another project caused by a range, as some packages may load different versions.

Reply:
Thanks Linnea, I just made a change to tighten down the ranges, but I think keeping them is a good idea in order to include point releases in within the one version we are on now, in this case 20. Please give the changes a re-review with you thoughts. Thanks!

Reply:
Removed the range